e2e/snapshot-creators/default.cy.snap.js

```diff
@@ -361,6 +361,9 @@ describe("snapshots", () => {
 });
 
 function getDefaultInstanceData() {
+  cy.request("PUT", "/api/setting", {
```

Comment:
We do this to ensure that the "All External Users" collection is returned from "/api/permissions/groups"